bdleon/nashville-sell-and-trade-client
# Bd nav css

#### Changes Made

1. Modified file `Login.js` to include `logo` and funtionality
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bd-nav-css
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    >check if logo appears in navbar
    > check if login is center with margin and check register as well with same check categories
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
